tests/code/test_transcripts.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from peewee import DoesNotExist
+
+from app.controllers.events.view_transcript import getSLCourseTranscript
+from app.controllers.events.view_transcript import getProgramTranscript
+from app.models.term import Term
+from app.models.course import Course
+from app.models.courseParticipant import CourseParticipant
+from app.models.user import User
+from app.models.eventParticipant import EventParticipant
+
+@pytest.mark.integration
+def test_noGetSLTranscript():
+
+    with pytest.raises(DoesNotExist):
+        sLcourse = getSLCourseTranscript(User.get_by_id("ggiuo"))
+    with pytest.raises(DoesNotExist):
+        sLcourse = getSLCourseTranscript(User.get_by_id(69809))
+
+@pytest.mark.integration
+def test_noGetProgramTranscript():
+
+    with pytest.raises(DoesNotExist):
+        sLcourse = getProgramTranscript(User.get_by_id("adfafa"))
+    with pytest.raises(DoesNotExist):
+        sLcourse = getProgramTranscript(User.get_by_id(56498))
+
+@pytest.mark.integration
+def test_getSLTranscripts():
+    user = User.get_by_id("neillz")
+    transcript = getSLCourseTranscript(user)
+
+    assert transcript[0][0]['fullName'] == ["Zach Neill"]

```

Comment:
I'm a little confused by the double array index here. Isn't it just one list of courses? Why would it return each field in its own dictionary that is in it's own list item?

Reply:
We used double array because we were testing to see if we were able to get the pieces of information (ex: course_name, hours ...) from our query. But we now got rid of the nested lists and just have one list for the whole transcripts so our code looks cleaner.